cmd/doctor.go

```diff
@@ -42,21 +51,30 @@ var checklist = []check{
 }
 
 func runDoctor(ctx *cli.Context) error {
-	err := initDB()
-	fmt.Println("Using app.ini at", setting.CustomConf)
-	if err != nil {
-		fmt.Println(err)
-		fmt.Println("Check if you are using the right config file. You can use a --config directive to specify one.")
-		return nil
-	}
+
+	// Silence the console logger
+	// TODO: Redirect all logs into `doctor.log` ignoring any other log configuration
+	log.DelNamedLogger("console")
+	log.DelNamedLogger(log.DEFAULT)
 
 	for i, check := range checklist {
+		if i == 1 {
```

Comment:
Just a passing comment, "magic" indexes like this are sometimes missed. I don't have a real suggestion at the moment, just wanted to make note of it.

Reply:
> 
> 
> Just a passing comment, "magic" indexes like this are sometimes missed.
Done.